assets/js/application.js

```diff
@@ -593,26 +593,39 @@ var Downloads = {
     return `<strong>${ago}</strong>, `;
   },
 
-  adjustFor32BitWindows: function() {
-    // adjust the auto-link for Windows 32-bit setups
-    const is32BitWindows = window.session.browser.os === 'Windows'
-      && !navigator.userAgent.match(/WOW64|Win64|x64|x86_64/)
-    if (!is32BitWindows) return;
-
-    const link = $('#auto-download-link');
-    const version = $('#auto-download-version');
-    const bitness = $('#auto-download-bitness');
-    const date = $('#auto-download-date');
-    if (link.length && version.length && bitness.length && date.length) {
-      bitness.html('32-bit');
-      link.attr('href', '{{ .Site.Params.windows_installer.installer32.url }}');
-      version.html('{{ .Site.Params.windows_installer.installer32.version }}');
-      date.html('{{ .Site.Params.windows_installer.installer32.release_date }}');
-    }
+  adjustForWindowsARM64: function() {
+    /*
+     * Windows/ARM64 cannot be reliably detected via the User-Agent string;
+     * Instead, we use the UAData, but that is not available in all browsers.
+     * For more details, see
+     * https://github.com/git-for-windows/git-for-windows.github.io/pull/61
+     */
+    if (!navigator.userAgentData) return;
+
+    navigator.userAgentData.getHighEntropyValues(['architecture', 'platform', 'bitness'])
+		  .then(function(browser) {
+				if (
+          browser.platform !== 'Windows'
+          || browser.bitness !== '64'
+          || browser.architecture !== 'arm'
+        ) return;
+
+        // adjust the auto-link for Windows/ARM64 setups
+        const link = $('#auto-download-link');
+        const version = $('#auto-download-version');
+        const architecture = $('#auto-download-architecture');
+        const date = $('#auto-download-date');
+        if (link.length && version.length && bitness.length && date.length) {
```

Comment:
```suggestion
        if (link.length && version.length && browser.bitness.length && date.length) {
```

Reply:
Oh, you're right, this must be the problem. But it should be changed to `architecture` instead of `browser.bitness`. Hold on a sec.

Reply:
Can confirm that it now correctly shows all ARM64 links. Thanks! 🎉 

![image](https://github.com/user-attachments/assets/47d95b5e-3a29-47fb-acf1-0a892c9d2936)

Reply:
@dennisameling could you have another look, please?

Reply:
Not sure if you saw my [comment](https://github.com/git/git-scm.com/pull/2001#discussion_r2068379474) from ~8 mins ago, but the ARM64 links are still showing up correctly as per your latest commit 👍🏼

Reply:
Oops. I had expected GitHub to update my browser page automatically, had not seen your answer! Thank you for confirming (twice!)!